examples/rag/README.md

```diff
@@ -0,0 +1,29 @@
+# Feast Quickstart
+If you haven't already, check out the quickstart guide on Feast's website (http://docs.feast.dev/quickstart), which 
```

Comment:
This README seems a generic quickstart guide with no specific connections with the RAG function or the included example.
I assume that it will be more clear once we run it and see what happens, but I believe than a dev experience driven by a notebook may help, WDYT?

Reply:
Let me know what you think!